data_generation/README.md

```diff
@@ -0,0 +1,37 @@
+# Generate data
+
+Use this scripts to generate data with the Inductiva API.
+
+The first thing that you need to do is export your api key as an
+environment variable:
+
+```bash
+export INDUCTIVA_API_KEY = <Your API Key>
+```
+
+This will prevent accidentally committing it to github since now we
+only need to do:
+
+```
+inductiva.api_key = os.environ['INDUCTIVA_API_KEY']
+```
+
+
+To submit the tasks use the script `submit_tasks.py`. This will submit
+the tasks to the API and save a `.txt` file with their IDs. To run the
+script you will `.vtk` objects over which to run the simulations. Once
+you have the `.vtk` objects over which to run the simulation run
+`python submit_tasks.py --input_dataset=<path to .vtk objects>`.
+
+To then fetch the output of these tasks you can run the script
+`download_tasks.py`. If no `--download_dir` is given the script will
```

Comment:
If the script is named download_tasks.py then it should download the data to a default bucket, no?

You could have a script to monitor things, if you desire.

Reply:
Yeah, I though about that, but then we need three scripts on our workflow. Where here we can just use 2 and then, when ready, download the data

Reply:
Will split in 3 scripts

Reply:
I believe the way to monitor things will be much more used, than the download outputs. Since simulations may take a long time, it is valuable to call it several times.